MODULE_REPORT.md

```diff
@@ -0,0 +1,10 @@
+## Module Report
```

Comment:
MODULE_REPORT.md probably shouldn't be included 😄

Reply:
I usually include it for housekeeping purposes. It avoids "why wasn't X converted" questions, and helps keep track of convertions that might be added to `ember-modules-codemod`.
We fixed the undefined bug in the report, I'll update ASAP.

Reply:
Awesome, thanks for explanation! 😊